### PR TITLE
docs: add scoop install instruction

### PR DESCRIPTION
**Pull request by AntonOks**
adding scoop to the "favorite package manager" list

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@lidel any reason not to merge this, despite Travis fail?

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Or you can use your favorite package manager:
 
 - **Homebrew** - `brew cask install ipfs`
 - **Chocolatey** - `choco install ipfs-desktop`
+- **Scoop** - `scoop install ipfs-desktop`
```
Comment (lidel) [inline README.md:91]: We need to point out this is a package maintained by a specific third party (see AUR below for example)

Comment (jessicaschilling) [reply README.md:91]: @AntonOks -- who is the maintainer for this package?

**Comment by jessicaschilling** [comment]
@AntonOks -- who is the maintainer for this package? If you can add that info we'll get this merged. Thank you!

File: README.md (hunk 1 above)
Comment (AntonOks) [reply README.md:91]: @jessicaschilling @lidel 
What do you mean with "maintainer for this package"? I don't know who initially added the scoop manifest for "ipfs-desktop" , but once it's there, it is "maintained by the scoop community"...
If it's about keeping ipfs-desktop up2date, scoop does this "automatically", see https://github.com/lukesampson/scoop-extras/blob/8de74e908a9e971d32dc5b3ee834e7494ade6738/bucket/ipfs-desktop.json#L17

Hope this answers your question?!

Comment (lidel) [reply README.md:91]: I see. It seems to be similar to homebrew, where [anyone can PR an update](https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask), but here it is [partially-automated](https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate). 

We should clearly state those packages are provided by the community on best-effort basis. 

@jessicaschilling  thoughts? Perhaps we could indicate that in header above the list:

> Or you can use your favorite package manager and a third party package maintained by the community:

Comment (jessicaschilling) [reply README.md:91]: SGTM!

**Comment by AntonOks** [comment]
@jessicaschilling @lidel THANKS both!

**Pull request merged**